### PR TITLE
refactor: unrepresented create claim confirmation screen (CMC-1245)

### PR DESCRIPTION
**Pull request by swalker125**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CMC-1245

### Change description ###

Added claim number and steps to proceed offline to confirmation screen

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandler.java [hunk 1]
```diff
@@ -65,8 +65,12 @@
         + "\n* Confirm service online within 21 days of sending the form, particulars and response pack, before"
         + " 4pm if you're doing this on the due day";
 
-    public static final String LIP_CONFIRMATION_BODY = "<br />You do not need to do anything.\n\n"
-        + "Your claim will be considered by the court and you will be informed of the outcome by post.";
+    public static final String LIP_CONFIRMATION_BODY = "<br />To continue your claim by post you need to:"
+        + "\n* [Download the sealed claim form](%s)"
+        + "\n* Send the claim form, <a href=\"%s\" target=\"_blank\">a response pack</a> (PDF, 266 KB) "
```
Comment (dharmendrak) [inline src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandler.java]: (PDF, 266 KB) does it need to hard coded

**Review by dharmendrak — DISMISSED** [review]
one observation

File: src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandler.java (hunk 1 above)
Comment (swalker125) [reply src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandler.java]: good question, it's currently hardcoded for other confirmation body. Will ask Steve in ticket

File: src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandler.java [hunk 2]
```diff
@@ -65,8 +65,12 @@
         + "\n* Confirm service online within 21 days of sending the form, particulars and response pack, before"
         + " 4pm if you're doing this on the due day";
 
-    public static final String LIP_CONFIRMATION_BODY = "<br />You do not need to do anything.\n\n"
-        + "Your claim will be considered by the court and you will be informed of the outcome by post.";
+    public static final String LIP_CONFIRMATION_BODY = "<br />To continue your claim by post you need to:"
+        + "\n* [Download the sealed claim form](%s)"
```
Comment (sabahirfan) [inline src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandler.java]: shouldn't we use `%n` instead of `\n` ?

**Review by sabahirfan — DISMISSED** [review]
One minor observation otherwise LGTM

**Review by sabahirfan — DISMISSED** [review]

**Pull request merged**